cmd/minishift/cmd/start_preflight.go

```diff
@@ -37,23 +37,38 @@ import (
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/viper"
 
-	cmdUtils "github.com/minishift/minishift/cmd/minishift/cmd/util"
+	cmdUtil "github.com/minishift/minishift/cmd/minishift/cmd/util"
 	minishiftNetwork "github.com/minishift/minishift/pkg/minishift/network"
 	openshiftVersion "github.com/minishift/minishift/pkg/minishift/openshift/version"
 	stringUtils "github.com/minishift/minishift/pkg/util/strings"
 )
 
 const (
 	StorageDisk   = "/mnt/?da1"
-	GithubAddress = "github.com:http"
+	GithubAddress = "https://github.com"
 )
 
 // preflightChecksBeforeStartingHost is executed before the startHost function.
 func preflightChecksBeforeStartingHost() {
 	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"
 	prerequisiteErrorMessage := "See the 'Installing Prerequisites for Minishift' topic (https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites) for more information"
 
+	proxy := os.Getenv("HTTPS_PROXY")
+
+	// In case we're not running with -v5, we want to obscure the proxy but
+	// still tell the user whether or not a proxy is being used. This is for
+	// security reasons as proxy URLs might contain credentials thay should not
+	// always be shown.
+	if !glog.V(5) {
+		if os.Getenv("HTTPS_PROXY") == "" {
+			proxy = "No"
+		} else {
+			proxy = "Yes"
+		}
+	}
+	fmt.Printf("-- Checking if %s is reachable (using proxy: %q) ... ", GithubAddress, proxy)
 	if network.CheckInternetConnectivity(GithubAddress) {
+		fmt.Printf("OK\n")
```

Comment:
you could use `fmt.Println("OK")` if newline is needed.

Reply:
I did it like this for consistency's sake as line 81 already does it using `fmt.Printf("\n")`.